src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -60,23 +60,27 @@
 
 (def ^:private ALL_USERS "All users")
 
+(defn- make-list [input]
+  ;;input may or maynot be a list, make it a list
+  (flatten (conj '() input)))
```

Comment:
Though, if the input really *was* `[1 [2 [3] 4] 5]`, `(sequential? )` would return true, and we'd attempt to pass the nested structure through ... resulting in bad data. So I agree that a generic `ensure-sequence` function should not flatten, but in this particular case we *do* want flatten.

I think it's a corner case so I don't feel strongly either way. Maybe we keep `ensure-sequence` generic, but we flatten in the location we call it from:
```
{:library:discoverableByGroups (flatten (make-list (get props :library:discoverableByGroups '())))}
```